doc/api/cluster.markdown

```diff
@@ -487,11 +487,29 @@ The `addressType` is one of:
 
 * `worker` {cluster.Worker}
 * `message` {Object}
+* `handle` {undefined|Object}
 
 Emitted when any worker receives a message.
 
 See [child_process event: 'message'][].
 
+Until Node.js v6.0, this event emitted only the message and the handle,
```

Comment:
Nit: this wording ('Until') makes it ambiguous what the behaviour of v6.0 is. Use 'Before Node.js v6.0' perhaps?